### ISSUE TITLE
Fails to stop vpnserver when active WireGuard sessions exist.

### ISSUE THREAD

**Issue by 1n5k** [SoftEtherVPN/SoftEtherVPN#1307]
### Prerequisites

* [x] Can you reproduce?
* [x] Are you running the latest version of SoftEtherVPN?

**SoftEther version:** 3761876254ae00e87e261f89910d093671a95109
**Component:** Server
**Operating system:** Linux
**Architecture:** 64 bit
**Processor:** Processor types don't matter

### Description

`vpnserver` cannot be stopped when active WireGuard sessions exist.

**Expected behavior:**
When `vpnserver stop` command is performed, all of active VPN sessions should be disconnected and then vpnserver should successfully be stopped.

**Actual behavior:**

***When no active WireGuard sessions, it can be stopped properly.***

```
root@sevpn:~/SoftEtherVPN/build# ./vpncmd /server /csv localhost:443 /hub:HAW /cmd:sessionlist
Session Name,VLAN ID,Location,User Name,Source Host Name,TCP Connections,Transfer Bytes,Transfer Packets
SID-SECURENAT-1,-,SecureNAT Session,SecureNAT,Virtual Host,None,862,16
```

```
root@sevpn:~/SoftEtherVPN/build# ./vpnserver stop
Stopping the SoftEther VPN Server service ...
SoftEther VPN Server service has been stopped.
```

***When active WireGuard sessions exist, it cannot be stopped and finally killed.***

```
root@sevpn:~/SoftEtherVPN/build# ./vpncmd /server /csv localhost:443 /hub:HAW /cmd:sessionlist
Session Name,VLAN ID,Location,User Name,Source Host Name,TCP Connections,Transfer Bytes,Transfer Packets
SID-SECURENAT-1,-,SecureNAT Session,SecureNAT,Virtual Host,None,"312,562","6,012"
SID-NAKASHIO-[WIREGUARD]-4,-,Local Session,nakashio,nakashio.lxd,1 / 1,"291,638","5,641"
SID-IWAO-[WIREGUARD]-6,-,Local Session,iwao,iwao.lxd,1 / 1,"274,664","5,294"
```
```
root@sevpn:~/SoftEtherVPN/build# ./vpnserver stop
Stopping the SoftEther VPN Server service ...
Stopping SoftEther VPN Server service was failed.
Killed
```

### Steps to reproduce

1. Setup vpnserver to accept WireGuard connections
2. Connect WireGuard clients
3. Confirm you can see active WireGuard sessions in SessionList
4. Perform `vpnserver stop` 
5. `Stopping SoftEther VPN Server service was failed.` will happen

**Comment by metalefty** [comment]
Hi, the reporter is my co-worker. I performed further tests. It properly shutdown even when the native protocol client is connected. However, same with WireGuard when L2TP/IPsec client is connected.

## Native protocol

```
ubuntu@ip-172-26-11-222:~/SoftEtherVPN/build$ ./vpncmd /server /csv localhost /hub:default /cmd:sessionlist
Session Name,VLAN ID,Location,User Name,Source Host Name,TCP Connections,Transfer Bytes,Transfer Packets
SID-SECURENAT-1,-,SecureNAT Session,SecureNAT,Virtual Host,None,"202,806",844
SID-IWAO-2,-,Local Session,iwao,c194201.ppp.asahi-net.or.jp,2 / 2,"208,108",868
```

```
ubuntu@ip-172-26-11-222:~/SoftEtherVPN/build$ sudo ./vpnserver stop
Stopping the SoftEther VPN Server service ...
SoftEther VPN Server service has been stopped.
```

## L2TP/IPsec 

```
ubuntu@ip-172-26-11-222:~/SoftEtherVPN/build$ ./vpncmd /server /csv localhost /hub:default /cmd:sessionlist
Session Name,VLAN ID,Location,User Name,Source Host Name,TCP Connections,Transfer Bytes,Transfer Packets
SID-SECURENAT-1,-,SecureNAT Session,SecureNAT,Virtual Host,None,"17,394",169
SID-IWAO-[L2TP]-2,-,Local Session,iwao,c194201.ppp.asahi-net.or.jp,1 / 1,"15,130",124
```

```
ubuntu@ip-172-26-11-222:~/SoftEtherVPN/build$ sudo ./vpnserver stop
Stopping the SoftEther VPN Server service ...
Stopping SoftEther VPN Server service was failed.
Killed
```

**Comment by metalefty** [comment]
I’ll have a test if #1427 solved this issue today.

**Comment by metalefty** [comment]
I confirmed the issue is fixed after #1427.

**Comment by davidebeatrici** [comment]
Thanks for confirming!
